waves/project.py

```diff
@@ -2559,3 +2559,96 @@ def generate_report(
         results_df.index = pd.Index([simulation_name])
         results_df.index.name = "Project"
         return results_df
+
+
+    def generate_report_lcoe_breakdown(self) -> pd.DataFrame:
+        """Generates a single dataframe of all the desired LCOE metrics from the project."""
```

Comment:
Could you add a little more context about this producing COWER/ATB reporting tables so it's a bit more clear for why it's doing the breakdowns it's doing? This could be as simple as making the one sentence a little longer. If it's also just a standard reporting breakdown, then that could be added instead of anything relating to NREL's reporting.

Reply:
"This function generates a single dataframe containing all the necessary LCOE metrics from the project, which are required to produce LCOE waterfall charts and CapEx donut charts for the Cost of Wind Energy Review. The dataframe provides a detailed breakdown, showing the contribution of each CapEx and OpEx component (from ORBIT and WOMBAT) to the LCOE in $/MWh."